test/unit/output/epacems_test.py

```diff
@@ -1,14 +1,37 @@
 """Test helper functions associated with the EPA CEMS outputs."""
 
 import logging
+from unittest.mock import patch
 
+import pandas as pd
 import pytest
 
-from pudl.output.epacems import year_state_filter
+from pudl.output.epacems import get_plant_states, get_plant_years, year_state_filter
 
 logger = logging.getLogger(__name__)
 
 
+# Test data containing plants across multiple states and years
+TEST_PLANT_DATA = pd.DataFrame(
```

Comment:
suuuper nit: I found myself transposing this in my head into a row-oriented format while reading the tests, could be a tiny test readability improvement to go to a list of records instead of a dict of columns. But it could also be an artifact of how my specific brain works.

Apart from that, yay! These tests are clear and illustrate important behavior!

Reply:
Oh yeah, I'll use the `io.StringIO` thing to make it an inline CSV.